specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -1118,6 +1246,90 @@
                     "description": "The size (%) of the segment."
                 }
             }
+        },
+        "SettingsList": {
+            "type": "object",
+            "description": "Subscription settings list.",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "description": "The settings list.",
+                    "items": {
+                        "$ref": "#/definitions/Settings"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "Settings": {
+            "type": "object",
+            "description": "Represents a security setting in Azure Security Center.",
+            "discriminator": "kind",
+            "properties": {},
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                },
+                {
+                    "$ref": "#/definitions/SettingsKind"
+                }
+            ]
+        },
+        "DataExportSettings": {
+            "type": "object",
+            "description": "Represents a data export setting",
+            "x-ms-discriminator-value": "DataExportSettings",
+            "properties": {
+                "properties": {
+				    "x-ms-client-flatten": true,
+                    "description": "Data export setting data",
+                    "$ref": "#/definitions/DataExportSettingsProperties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Settings"
+                }
+            ]
+        },
+        "SettingsKind": {
+            "type": "object",
+            "description": "The kind of the security settings",
+            "properties": {
+                "kind": {
+                    "type": "string",
+                    "description": "the kind of the settings string",
+                    "enum": [
+                        "DataExportSettings"
+                    ],
+                    "x-ms-enum": {
+                        "name": "SettingsKind",
+                        "modelAsString": true,
+                        "values": [
+                            {
+                                "value": "DataExportSettings"

```

Comment:
>DataExportSettings [](start = 42, length = 18)

can also be DataExport since you are in the context of a setting

Reply:
Changes all single settings to setting